app/views/sufia/admin/_sidebar.html.erb

```diff
@@ -0,0 +1,40 @@
+<ul class="admin-sidebar nav nav-pills nav-stacked">
```

Comment:
This is overriding the entire .html from sufia just to add the link to the new page?  That is fine (and probably the only way to do it), but add a comment to that effect, as well as a github URL to the exact version of the original sufia file you copied from, please and thank you!

Reply:
Correct! Will do, no problem at all.